test/serializer/optimized-functions/LoopBailout18.js

```diff
@@ -1,6 +1,6 @@
 function fn(x) {
   for (
-    var _iterator = x.items,
+    var _iterator = x.entries(),
```

Comment:
Why this change in an existing test?

Reply:
I just noticed that the test is actually broken and the intent is very clearly x.entries().